app/services/subcase-service.js

```diff
@@ -59,4 +59,18 @@ export default class SubcaseService extends Service {
     }
     return false;
   }
+
+  async getLatestDecisionActivity(subcase) {
```

Comment:
FYI, this is throwing some errors in jenkins because we don't have optional ? anywhere if the query return nothing or decision activity is not propagated yet. Suggested queries should resolve that.